Sources/Views/AnimatedImageView.swift

```diff
@@ -162,6 +173,101 @@ open class AnimatedImageView: UIImageView {
     // A flag to avoid invalidating the displayLink on deinit if it was never created, because displayLink is so lazy.
     private var isDisplayLinkInitialized: Bool = false
     
+#if os(macOS)
+    class DisplayLink {
```

Comment:
Can we move this class to another new file to keep this file a bit cleaner?

Reply:
Done